app/assets/stylesheets/items.scss

```diff
@@ -0,0 +1,63 @@
+.item {
+    font-size: 16px;
+    font-weight: bolder;
+    text-align: center;
+    margin-top: 15px;
+    z-index: 1;
+}
+
+.category {
+    font-size: 14px;
+    text-align: center;
+    color: gray;
+}
+
+.price {
+    color: $primary-color;
+    font-weight: bold;
+    text-align: center;
+}
+
+.menu-image {
+    width: 100%;
+    img {
+        border-radius: 50%;
+        width: 200px;
+        height: 200px;
+        overflow: hidden;
+        margin-bottom: 20px;
+        margin-top: 20px;
+        display: block;
+        margin-left: auto;
+        margin-right: auto;
+    }
+}
+.wrapper {
+    margin-right: auto;
+    margin-left: auto;
+    margin-top: 50px;
+    font-family: $font;
+    margin-bottom: 200px;
+}
+.row {
+    .col-lg-3, .col-md-4, .col-sm-6:not(:nth-of-type(4n+0)) {
```

Comment:
wow...are you sure you want this? 

keep in mind that this is change your columns for ALL the website. This is 100% not ok. Never change bootstrap base classes. Add additional classes in your views and style them.